File: SPOJ/AE00.py
n=int(input())
if(n==1):
    print(n)
else:
    count=0
    for i in range(1, (n//2)+1):
        for j in range(i, n+1):
            if(i*j<=n):
                count=count+1
            else:
                break
    print(count)

File: SPOJ/AE00.cpp
#include<iostream>
#include<math.h>
using namespace std;

int main()
{
  int N,K;
  long long count = 0;
  cin>>N;
  for(int i=1; i<=(int)sqrt(N); i++)
  {
    if(N%i == 0)
    {
      count += (N/i) - i + 1;
      // cout<<i<<" "<<N/i<<endl;
    }
  }
  cout<<count;
  return 0;
}
